command/project_set.go

```diff
@@ -66,6 +68,14 @@ func (cmd *ProjectSet) DoRun(args []string) (err error) {
 		return HandleError(errors.New("Project not found, please check the project name. You can get a list of your projects by running `nerd project list`."))
 	}
 
+	var p populator.P
+	p = &populator.EnvPopulator{}
+	p.SetKubeConfigFile()
```

Comment:
This call doesn't seem to make sense without any arguments or return values, what is the effect? Shouldn't it be part of the constructor of EnvPopulator?